### ISSUE TITLE
[WAC] - "Gateway Users" receive continuous login request and "error connecting to localhost" when logging into WAC

### ISSUE THREAD

**Issue by vanniekerkjaco** [microsoft/Windows-Admin-Center-Ideas-and-Feedback#121]
**Gateway Version**: 1.3.2111.01001

**To Reproduce**
1. Assign Gateway Users domain group in WAC Portal. This group and it members does not have any permissions on the Gateway server OS itself as they are restricted users.
2. Log into the WAC portal using a Gateway user account.
3. WAC portal will load, and you will receive an immediate request to re-authenticate connecting to local host
4. Cancel the auth request gives this error:
5. Error: Connecting to remote server localhost failed with the following error message : Access is denied. For more information, see the about_Remote_Troubleshooting Help topic.
6. Connecting to an endpoint to which the user does have Administrator access, fails with repeated auth requests

**Expected behavior**
Gateway users should not require Administrator or Remote Access Users local group permission on the Gateway server OS to access the Admin Center and other endpoints

**Screenshots & Additional context**
![image](https://user-images.githubusercontent.com/97897970/149950427-64b97fb2-a7c4-4258-be04-bbda072f33ce.png)
![image](https://user-images.githubusercontent.com/97897970/149950467-e0cccb0f-d022-4ad8-9116-b511c420518f.png)
![image](https://user-images.githubusercontent.com/97897970/149951447-24680b8a-8aeb-4af3-85f5-757d6f681a24.png)

**Comment by lbruck95** [comment]
I am seeing the same issue with some users.

**Comment by prasidharora** [comment]
@vanniekerkjaco @lbruck95 Can you confirm something for me? Are you saying that you have given access to WAC at the "Gateway Users" level, but access has not been given at the server OS level?

Giving access to WAC at the "Gateway Users" level gives access to that **instance of WAC** and **only that instance of WAC**. It **does not** give access to manage the gateway OS itself or any of the managed nodes. You would need valid admin credentials for the nodes themselves.

**Comment by lbruck95** [comment]
@prasidharora Part of the problem is that I can't see the Allowed Groups anymore. 
![Screenshot 2022-02-16 152735](https://user-images.githubusercontent.com/57728815/154360047-041dc846-9477-47c1-8d82-8607e4bb4534.png)
I have a group of users that I wanted to grant access to WAC but I can't even confirm that they are Gateway Users because of this. My second issue is that I wanted to give them access to view Active Directory on one of our Domain Controllers using WAC. Obviously, I don't want them to be Administrators on a DC and that isn't possible anyway. But when I give those users a link to access a shared connection to a DC in WAC, they login initially and then are prompted to login over and over again.

**Comment by vanniekerkjaco** [comment]
> @vanniekerkjaco @lbruck95 Can you confirm something for me? Are you saying that you have given access to WAC at the "Gateway Users" level, but access has not been given at the server OS level?

@prasidharora 
This only has to do with the WAC server itself, even before you try to connect to another node.
When a user is assigned the WAC gateway permission (which I have done in build 1.3.2105.24004, and then upgraded to build 2111), when that user logs into the wac portal and they are shown the "All Connections" window, they are presented with another login prompt to authenticate against "localhost". In this case localhost is the wac server itself. This login prompt will continue to popup endlessly even when you specify the valid "Gateway User" credential.
When I added the "Gateway User" account, to the WAC server OS Local Administrators or Local Remote Access Users group, they did not recieve the "localhost" login prompt anymore

**Comment by vanniekerkjaco** [comment]
> @prasidharora Part of the problem is that I can't see the Allowed Groups anymore. ![Screenshot 2022-02-16 152735](https://user-images.githubusercontent.com/57728815/154360047-041dc846-9477-47c1-8d82-8607e4bb4534.png) I have a group of users that I wanted to grant access to WAC but I can't even confirm that they are Gateway Users because of this. My second issue is that I wanted to give them access to view Active Directory on one of our Domain Controllers using WAC. Obviously, I don't want them to be Administrators on a DC and that isn't possible anyway. But when I give those users a link to access a shared connection to a DC in WAC, they login initially and then are prompted to login over and over again.

@lbruck95 Yes build 2111 has broken the ability to add groups to the WAC portal. I have downgraded to build 1.3.2105.24004 where I am able to manage this.

**Comment by ALZDBA** [comment]
uninstalling this mess as a simple logoff is not supported when you get this error

**Comment by davannaw-msft** [comment]
We've made a lot of improvements in Windows Admin Center since this issue was raised. Please the most recent version a try and let us know if the issue persists.
